ADOPTERS.md

```diff
@@ -0,0 +1,14 @@
+# Octant Adopters
+
+## Success Stories
+
+Below is a list of adopters of Octant in **production environments** that have
+publicly shared the details of how they use it.
+
+## Solutions built with Octant
```

Comment:
Solutions built with Octant would be plugins? .. This language is a bit confusing to me. There will probably never be very many solutions where Octant is being used at a component .. maybe some of the libraries we spin out of Octant eventually would fit that bill.

Reply:
I think plugins make sense here, for sure